src/core/tests/pattern_output_index.cpp

```diff
@@ -0,0 +1,205 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include "common_test_utils/graph_comparator.hpp"
+#include "common_test_utils/test_tools.hpp"
+#include "openvino/op/add.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/parameter.hpp"
+#include "openvino/op/relu.hpp"
+#include "openvino/op/reshape.hpp"
+#include "openvino/op/result.hpp"
+#include "openvino/op/split.hpp"
+#include "openvino/op/variadic_split.hpp"
+#include "openvino/pass/pattern/matcher.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+
+using namespace ov;
+using namespace ov::pass;
```

Comment:
Consider to wrap all test in:
`namespace ov::test{`

then using namespace should not be required and maybe just use ov::pass inside this namespace.